### PR TITLE
Fixed tab list dark mode design when tab screenshot is missing #698

### LINKED ISSUE #698: Fix tab list dark mode design when tab screenshot is missing
In some cases the website sceenshots for tab preview are missing. In such case tab list looks kinda broken in dark mode.

IMG_5354

Changing background tab color should work just fine.

### PR DESCRIPTION
**Pull request by shahenAnt**
Fix #698 

## Checklist:

- [ ] My squashed commit messages follows the [7 golden rules](https://chris.beams.io/posts/git-commit/)
- [ ] I updated or created necessary unit tests
- [ ] I've tested my changes in Dark Mode, iPad, iOS <= 12, Landscape and in German

### CODE REVIEW

File: UserAgent.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -6429,7 +6429,7 @@
 			baseConfigurationReference = 8F87D0B7200D772460EF6953 /* Pods-Shared.debug.xcconfig */;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
-				APPLICATION_EXTENSION_API_ONLY = YES;
+				APPLICATION_EXTENSION_API_ONLY = NO;
```
Comment (chrmod) [inline UserAgent.xcodeproj/project.pbxproj]: Hope this does not break Today widget

Comment (shahenAnt) [reply UserAgent.xcodeproj/project.pbxproj]: I have changed other targets settings to not required api only for extensions. And didn't touch to Today widget.

Comment (chrmod) [reply UserAgent.xcodeproj/project.pbxproj]: Awesome!

**Comment by chrmod** [comment]
Have run CI by hand and it looks good https://dev.azure.com/cliqz-ci/user-agent-ios/_build/results?buildId=3602&view=results

**Pull request merged**